batcher.js

```diff
@@ -13,22 +13,22 @@ if (process.argv.length < 4) {
 
 var command = process.argv[2];
 var query = process.argv[3];
-var data = parseData(process.argv[4]);
+var searchParams = sToO(process.argv[4]);
+var data = sToO(process.argv[5]);
 
-function parseData(data) {
-    // FIXME: not particularly typesafe.
-    try {
-        return JSON.parse(data)
-    } catch(e) {
-        return undefined;
+function sToO(s) {
+    // Wow!
+    eval('const o = '+ s);
```

Comment:
Done - I forgot you can pass arguments in as single quotes, making the call less :neckbeard:y.
e.g. 
`node batcher.js reindex "credit:'Bloomberg via Getty Images'" '{ "costModelDiff": true, "free": true, "missingIdentifier": "picdarUri" }' '{ "data": "bass" }'`